README.md

```diff
@@ -16,7 +16,7 @@ By default, `kcp` only knows about:
 
 Like vanilla Kubernetes, `kcp` persists these resources in etcd for durable storage.
 
-Any other resources, including Kubernetes-standard resources like [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)s, [`Node`](https://kubernetes.io/docs/concepts/architecture/nodes/)s and the rest, can be added as [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s and reconciled using the standard controllers.
```

Comment:
node is a bad example as we won't run the node controller or scheduler, and syncing (cluster-scoped) nodes doesn't make much sense.